docs/src/content/en/docs/evals/overview.mdx

```diff
@@ -23,6 +23,14 @@ There are different kinds of evals, each serving a specific purpose. Here are so
 2. **Classification Evals**: Measure accuracy in categorizing data based on predefined categories
 3. **Prompt Engineering Evals**: Explore impact of different instructions and input formats
 
+## Installation
```

Comment:
I followed this doc and ran into an error since I didn't have this package installed.